### DISCUSSION TITLE
阿里云盘挂载失败

### DISCUSSION THREAD

**Discussion by konley** [AlistGo/alist#2076] [Q&A]
![image](https://user-images.githubusercontent.com/45086581/197202522-82dcdf17-0244-4912-88c8-38c873ea6257.png)
显示：Post "https://auth.aliyundrive.com/v2/account/token": context deadline exceeded (Client.Timeout exceeded while awaiting headers)

**Comment by 12189108** [comment]
阿里云抽风，你会发现在服务器上也是ping不通的，过段时间自己会好

**Reply by ccbcfan** [reply]
跟楼主一样的出错提示，多次重试就没成功打开过。

**Reply by 12189108** [reply]
这没办法，可能ip被阿里云加入黑名单了？

**Reply by ccbcfan** [reply]
ip黑名单可能性不大，一是ip是动态的两天一换; 二是同ip下云盘app登录正常。

**Reply by byJoey** [reply]
同样的问题 并且可以ping通
root@OpenWrt:~# ping auth.aliyundrive.com
PING auth.aliyundrive.com (47.92.40.45): 56 data bytes
64 bytes from 47.92.40.45: seq=0 ttl=39 time=50.543 ms
64 bytes from 47.92.40.45: seq=1 ttl=39 time=50.331 ms
64 bytes from 47.92.40.45: seq=2 ttl=39 time=50.606 ms
64 bytes from 47.92.40.45: seq=3 ttl=39 time=50.349 ms
64 bytes from 47.92.40.45: seq=4 ttl=39 time=50.463 ms
64 bytes from 47.92.40.45: seq=5 ttl=39 time=50.469 ms
64 bytes from 47.92.40.45: seq=6 ttl=39 time=50.545 ms
64 bytes from 47.92.40.45: seq=7 ttl=39 time=50.506 ms
64 bytes from 47.92.40.45: seq=8 ttl=39 time=50.369 ms
64 bytes from 47.92.40.45: seq=9 ttl=39 time=50.433 ms

**Comment by byJoey** [comment]
问题解决 版本降级到v3.1.0-arm

**Comment by haoyana** [comment]
这个以后版本会修复吗
